daemon/backends/windows_proxy_controller.py

```diff
@@ -0,0 +1,64 @@
+import ctypes
+import importlib
+from daemon.backends._proxy_controller import ProxyController
+winreg_exists = importlib.util.find_spec('winreg')
```

Comment:
This is needed to avoid errors when this file is imported on macOS, because obviously `winreg` doesn't exist outside of Windows given that it's used to modify the Windows Registry.